android/assets/jsons/translations/template.properties

```diff
@@ -33,6 +37,8 @@ Requires [buildingName] to be built in the city =
 Requires [buildingName] to be built in all cities = 
 Provides a free [buildingName] in the city = 
 Requires worked [resource] near city = 
+# Translator note: Meaning "Requires at least one of the following resources worked near the city":
```

Comment:
Well then, just replace this with Requires at least one of the following resources worked near the city
It may be longer, but explicit is better, o?

Reply:
This too.